File: src/com/jda/algorithm_programs/Primes.java
/*******************************************************************
 * Purpose:
 *
 * @author Mohan Pednekar
 * @version 1.0
 * @since Jul 9, 2018
 *******************************************************************/
package com.jda.algorithm_programs;

import java.util.ArrayList;

import com.jda.utility.Util;

public class Primes {
	
	/**
	 * @param args
	 */
	public static void main(String[] args) {
		int maxPrime = 1000;
		ArrayList<Integer> primes = Util.primesUpto(maxPrime);
		primes.forEach(System.out::println);
	}
	
}

File: src/com/jda/algorithm_programs/Primes.kt
/*******************************************************************
 * Purpose:
 *
 * @author Mohan Pednekar
 * @version 1.0
 * @since Jul 9, 2018
 */
package com.jda.algorithm_programs

import com.jda.utility.Util
import java.util.function.Consumer

object Primes {

    /**
     * @param args
     */
    @JvmStatic
    fun main(args: Array<String>) {
        val maxPrime = 1000
        val primes = Util.primesUpto(maxPrime)
        primes.forEach(Consumer<Int> { println(it) })
    }
}
